system-tests/tests/go.mod

```diff
@@ -14,7 +14,7 @@ replace github.com/smartcontractkit/chainlink/deployment => ../../deployment
 replace github.com/smartcontractkit/chainlink/system-tests/lib => ../lib
 
 require (
-	github.com/ethereum/go-ethereum v1.15.7
+	github.com/ethereum/go-ethereum v1.16.1
```

Comment:
Is this unblocked? Previously we couldn't bump the patch version due to breaking changes: https://github.com/smartcontractkit/chainlink/pull/17928